### DISCUSSION TITLE
How to Reset Blocks in Grapesjs 0.21.1

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#5118] [Q&A]
![image](https://github.com/GrapesJS/grapesjs/assets/119593989/98f7efcb-6638-4e8c-8314-2ffbf19df83c)
**This the Old Method Which I was Used Before the Update Now it Shows the Property Doesn't Exist What is the New Type for this** 
`
editor.BlockManager.__destroy();
` 
**This is the New Methods ?**

**Comment by artf** [comment]
No, that should be the proper method. If it's what you get as a TS error, try to check the definition file and see if you notice any error.

**Reply by RakulAgn** [reply]
This is Only Method with the name of  `ComponentView.reset():` void Under Components Class and Under the Block Class There no reset Methods to call. I don't think so i am getting this properly. please give a glance at once right now i have used ts-ignore for the particular line
